server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -222,6 +242,10 @@ public DefaultConnectOptionsContext(ServiceConnectOptionsType connectOptions, Se
         }
 
         this.sslEncryptionEnabled = (sslEncryptionEnabled == null) ? true : sslEncryptionEnabled;
```

Comment:
Yeah, my vote if for final too. Might be worth changing the others to final to for consistency